### ISSUE TITLE
Pagination links in related HasMany 

### ISSUE THREAD

**Issue by kraionz** [laravel-json-api/laravel#84]
Hi, is there any way to show pagination links in related HasMany ?, just show meta.page {...}

Url api:
`<url>/products/<id>/releases?page[number]=2`

Response

``` json
{
  "meta": {
    "count": 20,
    "page": {
      "currentPage": 2,
      "from": 16,
      "lastPage": 2,
      "perPage": 15,
      "to": 20,
      "total": 20
     }
  },
"jsonapi": {},
"data": [...]
}
```

Thanks in advance

**Comment by lindyhopchris** [comment]
I believe this is a duplicate of #76 which is fixed in the next release. I'll try to tag that in the next few days.

**Comment by kraionz** [comment]
Hi, ok thank you very much I will be attentive to the next version

One more thing when generating the pagination is added "include =" without having any include, this is a Bug or it is by default of the JSON: API

**Comment by lindyhopchris** [comment]
Don't know, I'll have to look into that one. Again, it might already be fixed - I just don't know off the top of my head!!

**Comment by lindyhopchris** [comment]
Tagged the fix for this as `v1.0.0-beta.4`
